staging/src/k8s.io/cloud-provider/controllers/node/config/types.go

```diff
@@ -0,0 +1,23 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package config
+
+// NodeControllerConfiguration contains elements describing NodeController.
+type NodeControllerConfiguration struct {
+	// WorkerCount is the number of workers that are synchronizing nodes.
+	WorkerCount int32
```

Comment:
prefer the Concurrent*Syncs naming of the other controllers? 

```
git grep -e "Concurrent.*Syncs int32"
pkg/controller/cronjob/config/types.go: ConcurrentCronJobSyncs int32
pkg/controller/daemon/config/types.go:  ConcurrentDaemonSetSyncs int32
pkg/controller/deployment/config/types.go:      ConcurrentDeploymentSyncs int32
pkg/controller/endpoint/config/types.go:        ConcurrentEndpointSyncs int32
pkg/controller/endpointslice/config/types.go:   ConcurrentServiceEndpointSyncs int32
pkg/controller/endpointslicemirroring/config/types.go:  MirroringConcurrentServiceEndpointSyncs int32
pkg/controller/garbagecollector/config/types.go:        ConcurrentGCSyncs int32
pkg/controller/job/config/types.go:     ConcurrentJobSyncs int32
pkg/controller/namespace/config/types.go:       ConcurrentNamespaceSyncs int32
pkg/controller/replicaset/config/types.go:      ConcurrentRSSyncs int32
pkg/controller/replication/config/types.go:     ConcurrentRCSyncs int32
pkg/controller/resourcequota/config/types.go:   ConcurrentResourceQuotaSyncs int32
pkg/controller/serviceaccount/config/types.go:  ConcurrentSATokenSyncs int32
pkg/controller/statefulset/config/types.go:     ConcurrentStatefulSetSyncs int32
pkg/controller/ttlafterfinished/config/types.go:        ConcurrentTTLSyncs int32
pkg/controller/volume/ephemeral/config/types.go:        ConcurrentEphemeralVolumeSyncs int32
staging/src/k8s.io/cloud-provider/controllers/service/config/types.go:  ConcurrentServiceSyncs int32
staging/src/k8s.io/cloud-provider/controllers/service/config/v1alpha1/types.go: ConcurrentServiceSyncs int32
staging/src/k8s.io/kube-controller-manager/config/v1alpha1/types.go:    ConcurrentDaemonSetSyncs int32
staging/src/k8s.io/kube-controller-manager/config/v1alpha1/types.go:    ConcurrentDeploymentSyncs int32
staging/src/k8s.io/kube-controller-manager/config/v1alpha1/types.go:    ConcurrentStatefulSetSyncs int32
staging/src/k8s.io/kube-controller-manager/config/v1alpha1/types.go:    ConcurrentEndpointSyncs int32
staging/src/k8s.io/kube-controller-manager/config/v1alpha1/types.go:    ConcurrentServiceEndpointSyncs int32
staging/src/k8s.io/kube-controller-manager/config/v1alpha1/types.go:    MirroringConcurrentServiceEndpointSyncs int32
staging/src/k8s.io/kube-controller-manager/config/v1alpha1/types.go:    ConcurrentEphemeralVolumeSyncs int32
staging/src/k8s.io/kube-controller-manager/config/v1alpha1/types.go:    ConcurrentGCSyncs int32
staging/src/k8s.io/kube-controller-manager/config/v1alpha1/types.go:    ConcurrentJobSyncs int32
staging/src/k8s.io/kube-controller-manager/config/v1alpha1/types.go:    ConcurrentCronJobSyncs int32
staging/src/k8s.io/kube-controller-manager/config/v1alpha1/types.go:    ConcurrentNamespaceSyncs int32
staging/src/k8s.io/kube-controller-manager/config/v1alpha1/types.go:    ConcurrentRSSyncs int32
staging/src/k8s.io/kube-controller-manager/config/v1alpha1/types.go:    ConcurrentRCSyncs int32
staging/src/k8s.io/kube-controller-manager/config/v1alpha1/types.go:    ConcurrentResourceQuotaSyncs int32
staging/src/k8s.io/kube-controller-manager/config/v1alpha1/types.go:    ConcurrentSATokenSyncs int32
staging/src/k8s.io/kube-controller-manager/config/v1alpha1/types.go:    ConcurrentTTLSyncs int32
```

Reply:
Changed to ConcurrentNodeSyncs